WordPress/WordPressStatsWidgets/SiteListProvider.swift

```diff
@@ -39,8 +39,16 @@ struct SiteListProvider<T: HomeWidgetData>: IntentTimelineProvider {
     }
 
     func getTimeline(for configuration: SelectSiteIntent, in context: Context, completion: @escaping (Timeline<StatsWidgetEntry>) -> Void) {
+        guard let defaults = UserDefaults(suiteName: WPAppGroupName) else {
+            completion(Timeline(entries: [.noData(widgetKind)], policy: .never))
+            return
+        }
+        guard defaults.bool(forKey: AppConfiguration.Widget.Stats.userDefaultsJetpackFeaturesEnabledKey) else {
```

Comment:
One downside of checking for positive state is that if we add the `Widget` before the app launch it will immediately show the migration message. 

I think it could happen after the app update. If the app updates in the background as is not yet launched, the widgets will start showing this message even if the feature flag is not enabled. The same would happen for Jetpack app.

https://user-images.githubusercontent.com/4062343/215064025-ba3f949e-0d17-4727-b3b2-996152582369.mp4

Reply:
@staskus Nice catch!
Fixed in c9840a695f2193da109dd8112370f01a480be395 👍

Reply:
Thanks!